taskcluster/ac_taskgraph/build_config.py

```diff
@@ -26,7 +26,7 @@ def get_components():
     } for (name, project) in build_config['projects'].items()]
 
 
-def get_version(is_snapshot=False):
```

Comment:
I wonder what `is_snapshot` did 🤔

Reply:
I think it got lost along the way during previous cleanups.